libtransmission/file-win32.c

```diff
@@ -733,6 +733,23 @@ bool tr_sys_path_rename(char const* src_path, char const* dst_path, tr_error** e
     return ret;
 }
 
+bool tr_sys_path_copy(char const* src_path, char const* dst_path, uint64_t unused UNUSED, tr_error** error)
+{
+    DWORD const flags =
+#ifdef COPY_FILE_ALLOW_DECRYPTED_DESTINATION
```

Comment:
As with `COPYFILE_CLONE`, I'd define this macro with a fallback value at the top of the file and use it here without any `#if`-s.